### ISSUE TITLE
Local database: variants not always present in previous, highlighted cases

### ISSUE THREAD

**Issue by ielvers** [Clinical-Genomics/scout#2663]
When previous cases with the same variant are shown under local database, the previous cases are sometimes in grey:
![image](https://user-images.githubusercontent.com/57791644/119464084-b2315200-bd42-11eb-9e43-d5c3f45451af.png)
https://scout.scilifelab.se/cust003/21126/321f4cb4c1aea36b23f2e50813814b9d

In the grey cases, the variant cannot be found when searching for the gene name (sometimes other variants in the same gene are found). In a few instances we've checked by position in IGV and there is no trace of the variant. Why are these grey cases showing up?

**Comment by dnil** [comment]
Hi @ielvers! The cases in grey are ones where loqus informs there was an observation, but the variant could not be looked up by ID in the case in question. The usual reasons for this include missing variants due to reruns or variant purging of low ranking variants. A common one is also for SVs when the inexact matching of e.g. a large deletion from loqus returns a match, but we are unable to identify which one to the exact coordinate. This should give you a more accurate description of how many of the cases in the totals were actually from your institute. The change was introduced to correct issue #2580 through #2584. If you are interested in this particular variant, we can dig a bit into the VCFs and see why. We checked mostly SVs when developing this, so who knows. 😬

**Comment by dnil** [comment]
For your variant of interest, looking at the first grey match it was re-scored 4 for 19231 for the rerun and so not loaded the second time around (it scored a 10 on the first MIP 7 run).

**Comment by northwestwitch** [comment]
Perhaps we could add popup with a message saying something like: "Variant might have been removed after a rerun" when the link on the case is missing, for clarity sake..

**Comment by dnil** [comment]
Yes they do!

**Comment by ielvers** [comment]
Would it be possible to get a count without the grey cases? I guess that if it was an easy fix, you would already have implemented it. But since we only see the grey cases from our own institute we're suddenly very unsure of how to interpret the local observation count.

**Comment by dnil** [comment]
Do you think you could pinpoint where the uncertainty is?

It would be be possible, ideally to be dealt with in CG/loqus, but it is not totally out of the question just to count through the ones loaded in Scout. We sort of touch them anyway to check if we should display them or not, so its not much of a performance issue. 

But I'm not sure how different you would find it? The grayed out are for the most part just not loaded (due e.g. to changes in ranking or removing cases), or just for lack of a good system for bringing up links for almost similar SVs like a 2Mb deletion differing one base pair in start coordinate in Scout. Loqusdb does that much better.

**Comment by dnil** [comment]
Lets make this a discussion and see if we can get some others to weigh in!

**Comment by ielvers** [comment]
I take back all of my concerns, I was wrong :)
So when Helene and I started looking at these grey cases, for the first two variants we looked at we couldn't see them at all in IGV in the grey case. So we assumed that grey cases meant that the variant wasn't called anymore due to differential alignment of reads or smth. Hence our panicy reaction over them still being included in the local observation count. Today when I went through a larger number of variants with grey cases and looked in IGV in the grey cases, the variants were always present (where bam files were available) and when I went back to the first two variants we looked at last week, indeed they were present in the grey case too, I guess we just didn't scroll down to the parent. My bad!
